examples/dynamicCellProperty.cpp

```diff
@@ -0,0 +1,238 @@
+  Copyright 2017 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <examples/exampleSetup.hpp>
+
+#include <opm/utility/ECLCaseUtilities.hpp>
+#include <opm/utility/ECLPhaseIndex.hpp>
+#include <opm/utility/ECLPvtCommon.hpp>
+#include <opm/utility/ECLPvtCurveCollection.hpp>
+#include <opm/utility/ECLResultData.hpp>
+#include <opm/utility/ECLUnitHandling.hpp>
+
+#include <cstddef>
+#include <exception>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <boost/filesystem.hpp>
+
+namespace {
+    void openRestartSet(const Opm::ECLCaseUtilities::ResultSet& rset,
+                        const int                               step,
+                        std::unique_ptr<Opm::ECLRestartData>&   rstrt)
+    {
+        if (! (rset.isUnifiedRestart() && rstrt)) {
+            // Not a unified restart file or this is the first time we're
+            // seeing the result set.
+            rstrt.reset(new Opm::ECLRestartData(rset.restartFile(step)));
+        }
+    }
+}
+
+struct CellState
+{
+    CellState(const Opm::ECLGraph&                    G,
+              const Opm::ECLCaseUtilities::ResultSet& rset,
+              const int                               cellID_);
+
+    int                 cellID;
+    std::vector<double> time;
+    std::vector<double> Po;
+    std::vector<double> Rs;
+    std::vector<double> Rv;
+};
+
+CellState::CellState(const Opm::ECLGraph&                    G,
+                     const Opm::ECLCaseUtilities::ResultSet& rset,
+                     const int                               cellID_)
+    : cellID(cellID_)
+{
+    const auto u_press = &Opm::ECLUnits::UnitSystem::pressure;
+    const auto u_Rs    = &Opm::ECLUnits::UnitSystem::dissolvedGasOilRat;
+    const auto u_Rv    = &Opm::ECLUnits::UnitSystem::vaporisedOilGasRat;
+
+    const auto rsteps = rset.reportStepIDs();
+
+    this->time.reserve(rsteps.size());
+    this->Po  .reserve(rsteps.size());
+    this->Rs  .reserve(rsteps.size());
+    this->Rv  .reserve(rsteps.size());
+
+    auto rstrt = std::unique_ptr<Opm::ECLRestartData>{};
+
+    for (const auto& step : rsteps) {
+        openRestartSet(rset, step, rstrt);
+
+        rstrt->selectReportStep(step);
+
+        this->time.push_back(example::simulationTime(*rstrt));
+
+        {
+            const auto& Po =
```

Comment:
This shadows the Po member variable. Consider just `p` or `press`? Also: the variable names break the convention to start variable names with lower case.